ons_alpha/bulletins/models.py

```diff
@@ -14,11 +15,11 @@
     TabbedInterface,
 )
 from wagtail.contrib.routable_page.models import RoutablePageMixin, path
-from wagtail.models import Orderable, Page
+from wagtail.fields import StreamField
+from wagtail.models import Orderable, Page, Revision
 from wagtail.search import index
 
 from ons_alpha.core.models.base import BasePage
-from ons_alpha.utils.fields import StreamField
```

Comment:
please continue using this version of StreamField as it skips creating Django migrations for any StreamField block changes.

@RealOrangeOne this explains the migration.

Reply:
@RealOrangeOne  - are you happy to resolve this

Reply:
It looks like you're still using `wagtail.fields.StreamField` as opposed to `ons_alpha.utils.fields.StreamField`, so no this still needs doing.

Also, if the migrations it created aren't needed, those can be removed too.